src/Fixer/Import/NoUnusedImportsFixer.php

```diff
@@ -294,4 +302,20 @@ private function removeUsesInSameNamespace(Tokens $tokens, array $useDeclaration
             }
         }
     }
+
+    public function createConfigurationDefinition(): FixerConfigurationResolverInterface
+    {
+        return new FixerConfigurationResolver([
+            (new FixerOptionBuilder(self::CONFIG_OPTIPON_STATEMENTS, 'List of statements which must be removed.'))
```

Comment:
If you want to introduce this option in this PR, you should implement it and test it fully. For now only the `comment` value has effect and the option is not tested. To ease things, you can also introduce it in another PR and focus on `case_sensitive` here.

Reply:
Ok, I'll finish it as soon as possible